django_project/certification/views/certifying_organisation.py

```diff
@@ -217,6 +221,25 @@ def get_context_data(self, **kwargs):
         if project_slug:
             context['the_project'] = Project.objects.get(slug=project_slug)
             context['project'] = context['the_project']
+
+        if self.request.user.is_staff:
+            context['user_can_create'] = True
+            context['user_can_delete'] = True
+
+        certification_managers = (
+            context['the_project'].certification_managers.all())
+        if self.request.user in certification_managers:
```

Comment:
I just stumbled across this article, http://bookofstranger.com/optimizing-django-orm-queries-for-best-performance/

In that article, it's better to use `.exists()`. 

Another reference : https://docs.djangoproject.com/en/dev/ref/models/querysets/#exists

Reply:
Thanks @ismailsunni 
ill refactor that.

Reply:
Will you change also in the previous refactoring you did? And in the Lesson app in the worksheet view. Thanks @ismailsunni and @sonlinux

Reply:
Alright @Gustry 
Doing that just now

Reply:
I updated this the queryset @ismailsunni and @Gustry